content/id/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -0,0 +1,196 @@
+Local mempreservasi sumber IP client dan menghindari hop kedua _LoadBalancer_ dan servis tipe _NodePort_, namun
+resiko berpotensi penyebaran lalu lintas yang tidak merata.
+* `service.spec.healthCheckNodePort` - menentukan cek kesehatan _node_ porta (nomor porta numerik) untuk servis.
+Jika `healthCheckNodePort` tidak ditentukan, pengendali servis mengalokasi
+porta dari rentang _NodePort_ dari klaster kamu. Kamu dapat mengonfigurasi
+rentangan tersebut dari pengaturan opsi barisan perintah API server,
+`--service-node-port-range`. Hal itu menggunakan nilai `healthCheckNodePort` pengguna spesifik
+jika ditentukan oleh klien. Hal itu dapat berefek hanya ketika `type` diset ke _LoadBalancer_ dan
+`externalTrafficPolicy` diset ke Local.
+
+Pengaturan `externalTrafficPolicy` ke Local pada berkas konfigurasi Service mengaktifkan
+fitur ini.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-service
+spec:
+  selector:
+    app: example
+  ports:
+    - port: 8765
+      targetPort: 9376
+  externalTrafficPolicy: Local
+  type: LoadBalancer
+```
+
+## Pengumpul Sampah Load Balancers
+
+{{< feature-state for_k8s_version="v1.17" state="stable" >}}
+
+Pada kasus biasa, sumber daya _load balancer_ yang berkorelasi pada penyedia _cloud_ perlu
+dibersihkan segera setelah Service bertipe _LoadBalancer_ dihapus. Namun perlu diketahui
+bahwa terdapat kasus tepi dimana sumber daya _cloud_ yatim piatu (_orphaned_) setelah
+Service yang berkaitan dihapus. _Finalizer Protection_ untuk Service _LoadBalancer_
+diperkenalkan untuk mencegah hal ini terjadi. Dengan menggunakan _finalizers_, sebuah sumber daya Service
+tidak akan pernah dihapus hingga sumber daya _load balancer_ yang berkorelasi juga dihapus.
+
+Secara khusus, jika Service mempunyai `type` _LoadBalancer_, pengendali servis akan melekatkan
+_finalizer_ bernama `service.kubernetes.io/load-balancer-cleanup`.
+_Finalizer_ hanya akan dihapus setelah sumber daya _load balancer_ dibersihkan.
+Hal ini mencegah sumber daya _load balancer_ yang teruntai bahkan setelah kasus tepi seperti
+pengendali servis berhenti.
+
+## Penyedia Load Balancer Eksternal
+
+Penting untuk dicatat bahwa jalur data untuk fungsionalitas ini disediakan oleh _load balancer_ eksternal ke klaster Kubernetes.
+
+Ketika Service `type` diset `LoadBalancer`, Kubernetes menediakan fungsionalitas yang ekuivalen dengan `type` sebanding ClusterIP
+ke _pods_ dalam klaster dan mengekstensinya dengan pemrograman (eksternal ke Kubernetes) _load balancer_ dengan entri pada pods
+Kubernetes. Pengendali servis Kubernetes mengotomasi pembuatan _load balancer_ eksternal, cek kesehatan (jika dibutuhkan),
+dinding api (jika dibutuhkan), dan mengambil IP eksternal yang dialokasikan oleh penyedia _cloud_ dan mengisinya pada objek servis.
+
+## Peringatan dan and Limitasi ketika preservasi sumber IP
+
+_Load balancers_ GCE/AWS tidak menyediakan bobot pada kolam targetnya. Hal ini bukan merupakan isu dengan aturan kube-proxy
```

Comment:
better to use singular for imported words
```suggestion
_Load balancer_ GCE/AWS tidak menyediakan bobot pada kolom targetnya. Hal ini bukan merupakan isu dengan aturan kube-proxy
```

Reply:
Does it suppose to be `kolam`? It was referring to `target pools`, or should we leave it as `pools` instead? we're more familiar with `pool`